scanner/indexer/indexer_test.go

```diff
@@ -0,0 +1,54 @@
+package indexer
+
+import (
+	"testing"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_parseImageURL(t *testing.T) {
+	tests := []struct {
```

Comment:
can you add a case or two with digests as well?

Reply:
There is case below called `"with digest"`.